lib/opentok.js

```diff
@@ -1815,12 +1815,17 @@ OpenTok.prototype.dial = function (sessionId, token, sipUri, options, callback)
  * are optional):
  *
  * <ul>
- *
  *     <li><code>location</code> (String) &mdash;
  * An IP address that the OpenTok servers will use to situate the session in the global
  * OpenTok network. If you do not set a location hint, the OpenTok servers will be based on
  * the first client connecting to the session.
  * </li>
+ * 
+ *     <li><code>e2ee</code> (String[true|false]) &mdash;
```

Comment:
Do you really need to set this to a string (`'true'` or `'false'`)? Can you pass in a boolean (`true` or `false`)?

Reply:
as far as I know it has to be a string.

Reply:
I'm pretty sure that this is a Boolean.